src/config/config.prod.js

```diff
@@ -15,7 +15,7 @@ export default {
     },
   },
   signature: {
-    url: 'https://signatures.outcome-hub.com/signature',
+    url: 'https://signatures.outcome-hub.com',
```

Comment:
Is this correct? I remember it having to be /signature?

Reply:
Changed!

Reply:
Double check with @ngprnk, he may have changed the service.